File: 프로그래머스/1/12926. 시저 암호/시저 암호.js
function solution(s, n) {
    let answer = "";
    for (let i = 0; i < s.length; i++) {
        const char = s[i];
        if (char === " ") {
            answer += " ";
            continue;
        }

        const code = s.charCodeAt(i);
        if (code >= 65 && code <= 90) {
            answer += String.fromCharCode(((code - 65 + n) % 26) + 65);
        } else {
            answer += String.fromCharCode(((code - 97 + n) % 26) + 97)
        }
    }
    return answer;
}

File: 프로그래머스/1/12926. 시저 암호/시저 암호.py
def solution(s, n):
    answer = ''
    for char in s:
        if char == ' ':
            answer += ' '
        elif char.isupper():
            answer += chr((ord(char) - ord('A') + n) % 26 + ord('A'))
        elif char.islower():
            answer += chr((ord(char) - ord('a') + n) % 26 + ord('a'))
            pass
            
    return answer